core/src/main/java/com/novoda/landingstrip/animation/FastOutSlowInInterpolator.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.landingstrip.animation;
+
+class FastOutSlowInInterpolator extends LookupTableInterpolator {
```

Comment:
but what does that mean? Do you want to keep parity to update it? or is it adapted as our own and we clean it a little?